react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -180,6 +152,23 @@ export default {
     },
 
     /**
+     * The descriptor of the dial out toolbar button.
+     */
+    dialOut: {
+        classNames: [ 'button', 'icon-telephone' ],
+        enabled: true,
+
+        // Will be displayed once the SIP calls functionality is detected.
+        hidden: true,
+        id: 'toolbar_button_dial_out',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
+            APP.store.dispatch(openDialOutDialog());
```

Comment:
I had a chance of discussing this with Hristo off-github and we agreed that there are a lot of fixes to be done in the defaultToolbarButtons.js and this should be done as part of those fixes when we're ready to come back to the toolbox improvements.